docs/components/sidecar.md

```diff
@@ -70,7 +70,7 @@ config:
   bucket: example-bucket
 ```
 
-## Upload compacted blocks (EXPERIMENTAL)
```

Comment:
Is it non experimental? (:

Reply:
I updated the doc because I saw this pr https://github.com/thanos-io/thanos/pull/1833/files#diff-e44a45d13f9d7834ba8c09c23cfcec12. I thought this feature is already marked as ready